packs/elastic/actions/search.py

```diff
@@ -0,0 +1,89 @@
+from easydict import EasyDict
+from lib.items_selector import ItemsSelector
+from lib.esbase_action import ESBaseAction
+from collections import defaultdict
+import logging
+import sys
+import elasticsearch
+import json
+
+logger = logging.getLogger(__name__)
```

Comment:
I'd love to. Definitely logging should be reworked. But I don't know how yet...
There are some problems with curator, if it's possible to route it to use the same logger which is self.logger it would be cool.
Plus there are whitelists (or black) I don't really remember, which are also used in curator.